src/Filters/ActivityListSort.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Filters;
+
+use DateTime;
+
+class ActivityListSort
+{
+    /** @var DateTime */
+    public $createdAt;
+
+    /** @var DateTime */
+    public $finalDeadline;
+
+    /** @var string */
+    public $name;
+
+    public function setSortingFields($sorting)
+    {
+        if ($sorting) {
+            if (array_key_exists('name', $sorting)) {
+                $this->name = $sorting['name'];
```

Comment:
Validate what value is coming here. What if someone make a request `?sortby[name]=invalid_value` ?